{{cookiecutter.project_dashed}}-src/{{cookiecutter.project_namespace}}/views/public.py

```diff
@@ -36,4 +38,6 @@ def get(self):
         # something like Cronitor is hitting this URL repeatedly to monitor uptime.
         log.info('ping-db ok')
 
+        ctasks.ping.apply_async()
```

Comment:
This isn't right, you need to call ping_url, not just ping: https://github.com/level12/agari-adm/blob/master/agariadm/views/public.py#L23